### DISCUSSION TITLE
Releasing v2.3

### DISCUSSION THREAD

**Discussion by FObermaier** [NetTopologySuite/NetTopologySuite#512] [General]
The `deveopment` branch is in sync with JTS v1.18.1 release. 

Should we issue a new release? If so should it be 
- [ ] v2.2.1 _or_
- [ ] v2.3?

If we go for v2.3, we should at least resolve #508, maybe #482?

**Comment by airbreather** [comment]
2.3 makes sense to me.  I should be able to wrap up #482 and #508 this weekend.
